daemon/daemon_utils.go

```diff
@@ -16,6 +21,10 @@ var (
 	runtimeScriptPerm os.FileMode = 0700
 )
 
+const (
+	snapshotterPluginType = "io.containerd.snapshotter.v1"
```

Comment:
why doesn't use export `plugin. SnapshotPlugin` type here?

Reply:
Fix it! @fuweid